xbmc/addons/GUIDialogAddonInfo.cpp

```diff
@@ -86,6 +87,15 @@ bool CGUIDialogAddonInfo::OnMessage(CGUIMessage& message)
       }
       if (iControl == CONTROL_BTN_INSTALL)
       {
+        if (m_localAddon)
+        {
+          if (m_localAddon->Type() == ADDON_ADSPDLL && ActiveAE::CActiveAEDSP::Get().IsProcessing())
+          {
+            CGUIDialogOK::ShowAndGetInput(24077, 0, 24085, 0);
+            return true;
+          }
+        }
+
```

Comment:
Please see the CVariant fixes @Paxxi has merged as well for doing the same for this PR. Also is which variant of the yes/no dialog is used as we should only use the textbox input instead of seperate lines